frontend/src/components/AddForm.tsx

```diff
@@ -0,0 +1,69 @@
+import {FormEvent, useState} from "react";
+import {useNavigate} from "react-router-dom";
+import Button from "@mui/material/Button";
+
+
+type Props = {
+    onAddParty: (data: { date: string; location: string; theme: string }) => void;
+}
+
+export default function AddForm(props: Props) {
+
+    const [theme, setTheme] = useState("");
```

Comment:
You may want to use generic typing, to utilize TypeScript for useState hook

```suggestion
    const [theme, setTheme] = useState<string>("");
```

Reply:
Isn't the type already set when using a default value? The states are all shown as `: string` in IntelliJ

Reply:
You're right, IntelliJ is smart here, but imagine people on different IDEs. I don't actually know what's the best practice here, but I noticed that the mui code from the docs is indeed using generics for its useStates.

Reply:
Good to know! 💪